atd-vze/src/views/Tools/CreateCrashRecordTable.js

```diff
@@ -23,7 +23,10 @@ import {
 
 const GET_TEMP_RECORDS = gql`
   query getTempRecords {
-    crashes(where: { is_temp_record: { _eq: true } }) {
+    crashes(
+      where: { is_temp_record: { _eq: true } }
+      order_by: { record_locator: asc }
```

Comment:
was annoying me that the temp records table wasnt being sorted so it was hard to find ur last record creation

Reply:
do you want to sort this by `desc` instead, so the most recent records are at the top?

also, i could have sworn you had configured the query to refetch after form submit so that the temp records table gets refreshed. that's not happening anymore. maybe i broke that when i patched your branch?? 🙈